common/src/main/java/org/corfudb/common/metrics/micrometer/MeterRegistryProvider.java

```diff
@@ -181,7 +181,9 @@ public static void registerExternalSupplier(String identifier, Supplier<String>
     }
```

Comment:
Not using synchronize here or making the map Collections.synchronizedMap()  because we don't need it here.  Making this synchronous might have some performance delays on opening a table in disk_backed mode.

Reply:
I tested the time taken for the following line to execute and it ranges anywhere from 0ms to 11ms -> https://github.com/CorfuDB/CorfuDB/blob/master/common/src/main/java/org/corfudb/common/metrics/micrometer/registries/LoggingMeterRegistryWithHistogramSupport.java#L201